### ISSUE TITLE
Is this project dead ?

### ISSUE THREAD

**Issue by cbodden** [Boerderij/Varken#250]
I assume seeing as how there hasn't been any movement that it is a dead project but wanted to make sure so that i stop running it and worry about updates.

**Comment by zcmack** [comment]
also interested, no releases since 2020 and most of the installation doc is broken.

**Comment by Dragonsys** [comment]
No, it is just slow, and the Wiki is due a rebuild. You might want check out the Discord server.

**Comment by onedr0p** [comment]
I'm fairly certain this project is dead. @dirtycajunrice could probably confirm though.

**Comment by TATUMTOT** [comment]
Im still using it

**Comment by MDHMatt** [comment]
> I assume seeing as how there hasn't been any movement that it is a dead project but wanted to make sure so that i stop running it and worry about updates.

use the develop branch. its more up to date

**Comment by mholzma** [comment]
Does someone else have to take over maintenance so you can have a stable release versus the multiple development branches that people are using?
